### PR TITLE
docs: Update gstreamer package in demos

### PR DESCRIPTION
**Pull request by kate-goldenring**
**What this PR does / why we need it**:
As a follow up to #91, update gstreamer package in other documentation.

### CODE REVIEW

File: docs/end-to-end-demo-rpi4.md [hunk 1]
```diff
@@ -48,7 +48,7 @@ This demo will demonstrate how to get Akri working on a **Raspberry Pi 4**, all
     ```
 1. Install Gstreamer main packages
     ```sh
-    sudo apt-get install -y libgstreamer1.0-0 gstreamer1.0-dev gstreamer1.0-tools \
+    sudo apt-get install -y libgstreamer1.0-0 libgstreamer1.0-dev gstreamer1.0-tools \
                             gstreamer1.0-doc gstreamer1.0-plugins-base gstreamer1.0-plugins-good \
                             gstreamer1.0-plugins-bad gstreamer1.0-plugins-ugly \
                             gstreamer1.0-libav gstreamer1.0-doc gstreamer1.0-tools \
```
Comment (bfjelds) [inline docs/end-to-end-demo-rpi4.md:54]: while we're cleaning ... it looks like `gstreamer1.0-doc` in in here twice.

Comment (bfjelds) [reply docs/end-to-end-demo-rpi4.md:54]: (and gstreamer1.0-tools)

**Review by bfjelds — APPROVED** [review]

File: docs/end-to-end-demo-rpi4.md (hunk 1 above)
Comment (jiria) [reply docs/end-to-end-demo-rpi4.md:54]: Curious, why do we need the doc package?

Comment (kate-goldenring) [reply docs/end-to-end-demo-rpi4.md:54]: We may not need it. I can investigate

File: docs/end-to-end-demo-rpi4.md [hunk 2]
```diff
@@ -48,12 +48,9 @@ This demo will demonstrate how to get Akri working on a **Raspberry Pi 4**, all
     ```
 1. Install Gstreamer main packages
     ```sh
-    sudo apt-get install -y libgstreamer1.0-0 gstreamer1.0-dev gstreamer1.0-tools \
-                            gstreamer1.0-doc gstreamer1.0-plugins-base gstreamer1.0-plugins-good \
-                            gstreamer1.0-plugins-bad gstreamer1.0-plugins-ugly \
-                            gstreamer1.0-libav gstreamer1.0-doc gstreamer1.0-tools \
-                            gstreamer1.0-x gstreamer1.0-alsa gstreamer1.0-gl gstreamer1.0-gtk3 \
-                            gstreamer1.0-qt5 gstreamer1.0-pulseaudio 
+    sudo apt-get install -y \
```
Comment (kate-goldenring) [inline docs/end-to-end-demo-rpi4.md:51]: does this need to be indented more?

File: docs/end-to-end-demo-rpi4.md (hunk 1 above)
Comment (kate-goldenring) [reply docs/end-to-end-demo-rpi4.md]: a lot ended up not being needed in order to mock the videos, including the `libgstreamer1.0-dev` which prompted this PR

File: docs/end-to-end-demo-rpi4.md (hunk 2 above)
Comment (jiria) [reply docs/end-to-end-demo-rpi4.md:51]: nope

**Review by jiria — APPROVED** [review]

**Pull request merged**